www/source/community.html.slim

```diff
@@ -1,6 +1,8 @@
 ---
 title: InSpec - Community
-description: Interact with the InSpec community - contribute to the project, provide feedback, and ask questions of other community members.
+description: InSpec is an open source (OSS) automated testing tool for integration, compliance, security and other policy requirements. Contribute to the InSpec project, provide feedback, and ask questions of other community members.
```

Comment:
Since the Oxford comma is used in other places, can we add it here:

```
...for integration, compliance, security, and other policy requirements
```